pages/terms.module.css

```diff
@@ -0,0 +1,18 @@
+.card {
+  max-width: 80ch;
+  margin: 2rem auto;
+}
+
+/* In the following code .card must be removed  */
+.card section {
+  margin: 0 0 1.5rem;
+  counter-increment: section;
+}
+
+.card section > section {
+  counter-reset: section;
+}
+
+.card :matches(h2, h3, h4, h5, h6)::before {
+  content: counters(section, '.') '. ';
```

Comment:
Wow didn't know something like existed. 👍

Reply:
It's nice but I am struggling to properly configure it. I am afraid, I will need to create a counter per heading type because I cannot make sections to pick it up automatically.